src/question.js

```diff
@@ -0,0 +1,116 @@
+//declare a list of questions which will be prompted to the user
+const initialQuestions = [
```

Comment:
Consider adding validation onto the questions to ensure you receive the data that you need.